product_docs/docs/pgpool/1.0/01_installing_and_configuring_the_pgpool-II.mdx

```diff
@@ -25,7 +25,7 @@ Pgpool-II runs as a service on Linux systems. Windows systems does not support P
 | Pgpool 3.6         | EDB Postgres Advanced Server 9.6 and 10                 | RHEL 7 - x86_64 <br/>RHEL 7 - ppc64le <br/>Linux graphical installer                                                                                              |
 
 !!! Note
-    - Pgpool does not support CentOS/RHEL/OEL 6.x platforms. EDB recommends migrating EDB products running on CentOS/RHEL/OEL 6.x platforms to a supported platform.
+    - Pgpool does not support CentOS/RHEL/OL 6.x platforms. EDB recommends migrating EDB products running on CentOS/RHEL/OL 6.x platforms to a supported platform.
```

Comment:
Can we use the note similar to the one used above: "Pgpool is no longer supported..."

Reply:
Edited. Thanks